cp2k_output_tools/blocks/energies.py

```diff
@@ -13,9 +30,16 @@
 
 
 def match_energies(content: str) -> Optional[BlockMatch]:
+    spans = []
+    energies = {}
+    match = MAIN_ENERGY_RE.search(content)
+    if match:
+        energies = match.groupdict()
+        spans = match.spans(0)
     match = FORCE_EVAL_ENERGY_RE.search(content)
-
-    if not match:
+    if match:
+        energies["total force_eval"] = match["value"]
+        spans += match.spans(0)
```

Comment:
```suggestion
        spans += [match.spans(0)]
```

Reply:
I reversed this change since it was resulting in the cli tests to fail. Checking the output I believe that my original implementation was actually already giving the correct output?